css/home-page-redesign.styl

```diff
@@ -91,21 +133,34 @@ GRAY = #EFF2F5
       @media (max-width: 900px)
         width: 100%
 
-    h1, p, .standard-button
-      color: NAVY
+    .discover-button
+      border: 2px solid
+      border-radius: 6px
+      border-color: DARK-BLUE
+      background: none
+      padding: 0.75em 1.75em
+      transition: background 500ms ease
+      margin: 50px 10px 0 10px
+      font-size: 15px
+      font-weight: bold
+      text-decoration: none
+      color: DARK-BLUE
 
-    .standard-button
-      border: 4px solid NAVY
+      @media (max-width: 900px)
+        margin-top: 20px
 
       &:hover
-        background: rgba(0, 0, 0, 0.1)
+      &:active,
+      &:focus
+        border-color: GRAY
```

Comment:
By matching the border to the background, you're basically removing it.  It makes the button "bouncy."  I'd remove this and just leave it as `.discover-button{ border: 2px solid DARK-BLUE; }`.